### DISCUSSION TITLE
PHPMailer no funciona en mi servidor

### DISCUSSION THREAD

**Discussion by AntonioHC-Adfin** [PHPMailer/PHPMailer#2580] [Q&A]
En Localhost funciona perfectamente, pero cuando lo subo a mi servidor, no funciona, dice "ESTA PAGINA NO FUNCIONA" 
HTTP ERROR 500
La prueba que estoy haciendo es:

use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\Exception;
use PHPMailer\PHPMailer\SMTP;

require  'phpmailer/src/exception.php';
require  'phpmailer/src/phpmailer.php';
require  'phpmailer/src/smtp.php';

$mail = new PHPMailer(true);

try {

    //Server settings
    $mail->SMTPDebug = 0;
    $mail->isSMTP();
    $mail->Host       = 'smtp.gmail.com';              //Set the SMTP server to send through
    $mail->SMTPAuth   = true;                                   //Enable SMTP authentication
    $mail->Username   = 'micorreo@gmail.com';                     //SMTP username
    $mail->Password   = 'miclave';                               //SMTP password
    $mail->SMTPSecure = 'tls';
    $mail->Port       = 587;                                   

    $mail->setFrom('micoreo@gmail.com', 'Mi correo');
    $mail->addAddress('correodestino@gmail.com', 'Correo destino');   

    // $mail->addAttachment('nombre y ruta del documento')

    $mail->isHTML(true);
    $mail->Subject = 'Prueba de correo';
    $mail->Body = 'ESTO ES UNA PRUEBA DEL CUERPO-Usando Gmail';
    $mail->send();

    echo 'Correo enviado';
} catch (Exception $e) {
	echo 'Mensaje ' . $mail-> ErrorInfo;
}

?>

**Comment by Synchro** [comment]
Please read the troubleshooting guide. It's very likely your hosting provider is blocking outbound SMTP.

**Comment by AntonioHC-Adfin** [comment]
Gracias, el problema era que en el servidor usaba un PHP antiguo, lo he actualizado y solucionado.
